js/load.js

```diff
@@ -29,10 +29,97 @@ const work = {
+  image: {
+    src: `./images/snapshoot/snapshoot_${this.id}.svg`,
+    alt: 'Project image',
+  },
+  info: {
+    author: 'RAMOUN',
+    type: 'Back End Dev',
+    year: '2015',
+  },
+  technologies: ['html', 'css', 'JavaScript', 'cpp'],
+  buttons: {
+    live: {
+      text: 'see live',
+      link: '#',
+      imgSrc: './images/arrow.svg',
+    },
+    source: {
+      text: 'see source',
+      link: '#',
+      imgSrc: './images/social/github.svg',
+    },
+  },
+};
+const work4 = {
+  id: 4,
+  name: 'Tonic',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  fullDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  LongDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  image: {
+    src: `./images/snapshoot/snapshoot_${this.id}.svg`,
+    alt: 'Project image',
+  },
+  info: {
+    author: 'RAMOUN',
+    type: 'Back End Dev',
+    year: '2015',
+  },
+  technologies: ['html', 'css', 'JavaScript', 'cpp'],
+  buttons: {
+    live: {
+      text: 'see live',
+      link: '#',
+      imgSrc: './images/arrow.svg',
+    },
+    source: {
+      text: 'see source',
+      link: '#',
+      imgSrc: './images/social/github.svg',
+    },
+  },
+};
+const works = [work1, work2, work3, work4];
```

Comment:
Kindly make these requited changes:
- In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.

Kindly store your ` const ` inside of an ` array `:

```
const projects = [
 {
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
 },
  {  name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
   }
]

```

**_The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project._**

I hope that this information will be helpful for you.